CodeStarterCamp_Week2/main.swift

```diff
@@ -1,12 +1,26 @@
-//
-//  main.swift
-//  CodeStarterCamp_Week2
-//
-//  Created by yagom.
-//  Copyright © yagom academy. All rights reserved.
-//
+func makeWinningLottoNumbers() -> [Int] {
+    var winningLottoNumbers = Set<Int>()
+    while winningLottoNumbers.count < 6 {
+        winningLottoNumbers.insert(Int.random(in: 1...45))
+    }
+    return winningLottoNumbers.sorted()
+}
 
-import Foundation
-
-print("Hello, World!")
+func checkMyLottoNumbers(myLottoNumbers: [Int]) {
+    let winningLottoNumbers = makeWinningLottoNumbers()
+    print("이번 당첨번호: \(winningLottoNumbers)")
+    var sameNumbers: [Int] = []
+    
+    for nums in myLottoNumbers {
+        if winningLottoNumbers.contains(nums) == true {
```

Comment:
요건 사바사 이긴 합니다만, 저는 조건문의 true는 생략하는 편입니다.
어떤 분은 토비님 처럼 작성하는게 확실하게 명시해줘서 좋다고 말씀하시는 분도 계십니다!
요건 참고하시면 좋을거 같아 말씀 드려요~ 😁
```suggestion
        if winningLottoNumbers.contains(nums) {
```

Reply:
앗 그러네요 contains메소드자체가 Bool타입이라 없어도 되는거였군요...!!!